packages/particle-network/src/index.ts

```diff
@@ -0,0 +1,110 @@
+import {
+	WalletInit,
+	EIP1193Provider,
+	ProviderRpcError,
+	ProviderRpcErrorCode,
+	ProviderAccounts,
+} from '@web3-onboard/common';
+import { createEIP1193Provider } from '@web3-onboard/common';
+import { ParticleNetwork, Config } from '@particle-network/auth';
+import { ParticleProvider } from '@particle-network/provider';
+
+type AuthTypes =
+	| 'email'
+	| 'phone'
+	| 'google'
+	| 'apple'
+	| 'twitter'
+	| 'facebook'
+	| 'microsoft'
+	| 'linkedin'
+	| 'github'
+	| 'twitch'
+	| 'discord';
+
+interface PreferredAuthType {
+	type: AuthTypes;
+	setAsDisplay: boolean;
+}
+
+interface ParticleAuthModuleOptions extends Config {
+	preferredAuthType?: AuthTypes | PreferredAuthType;
+}
+
+const getDisplayLabel = (authType?: string, shouldSetDisplay?: boolean) => {
```

Comment:
Can you please run the prettier formatter when you have a moment on this file? This will allow the particle module to match the rest of the codebase in layout and allow for easier IDE readability 🤓

Reply:
Sure, just ran it through the formatter, should be good to go now